suite-native/state/src/store.ts

```diff
@@ -24,30 +27,27 @@ if (__DEV__) {
     middlewares.push(reduxFlipperDebugger);
 }
 
-export const transactionsReducer = prepareTransactionsReducer(extraDependencies);
-export const accountsReducer = prepareAccountsReducer(extraDependencies);
-export const fiatRatesReducer = prepareFiatRatesReducer(extraDependencies);
-export const blockchainReducer = prepareBlockchainReducer(extraDependencies);
+const rootPersistConfig = {
+    key: 'root',
+    storage: reduxStorage,
+    whitelist: ['appSettings', 'wallet'],
+};
 
-const walletReducers = combineReducers({
-    accounts: accountsReducer,
-    blockchain: blockchainReducer,
-    fiat: fiatRatesReducer,
-    transactions: transactionsReducer,
-});
+const persistedReducer = persistReducer(rootPersistConfig, rootReducers);
 
 export const store: Store = configureStore({
-    reducer: {
-        appSettings: appSettingsReducer,
-        devices: devicesReducer,
-        wallet: walletReducers,
-    },
+    reducer: persistedReducer,
     middleware: getDefaultMiddleware =>
         getDefaultMiddleware({
             thunk: {
                 extraArgument: extraDependencies,
             },
+            serializableCheck: {
+                ignoredActions: [FLUSH, REHYDRATE, PAUSE, PERSIST, PURGE, REGISTER],
+            },
         }).concat(middlewares),
 });
 
 export type RootState = ReturnType<typeof store.getState>;
+
+export const persistor = persistStore(store);
```

Comment:
`storePersistor`

Reply:
I suppose, since the prop is named `persistor ` it's ok?

Reply:
In global context it's better to know it's related to redux store.